scenarios/aws/vm/ec2vm/ec2VM.go

```diff
@@ -4,10 +4,11 @@ import (
 	"fmt"
 	"os"
 
-	commonos "github.com/DataDog/test-infra-definitions/components/os"
+	componentos "github.com/DataDog/test-infra-definitions/components/os"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why do you need an alias ?

Reply:
Because I have to import `os` for [os.ReadFile](https://pkg.go.dev/os#ReadFile)